src/mumble/mumble_sv.ts

```diff
@@ -5314,6 +5314,16 @@ Om inte, avbryt och kontrollera ditt certifikat eller användarnamn.</translatio
     </message>
```

Comment:
Sorry. Translations have to be done on Transifex. We can't accept translations via GitHub directly, so this will have to go.